dlp/system-test/deid.test.js

```diff
@@ -333,4 +335,29 @@ describe('deid', () => {
     }
     assert.include(output, 'INVALID_ARGUMENT');
   });
+
+  // dlp_deidentify_table_fpe
+  it.skip('should de-identify table using Format Preserving Encryption (FPE)', () => {
```

Comment:
Why are we skipping the tests in this file?

Reply:
They were skipped because we didn't have resources like `wrappedKey` and `keyName`. Anyway, now we are using mock API calls, so removed the `it.skip` now and updated the tests. Thanks